examples/quantum_volume.ipynb

```diff
@@ -1,5 +1,77 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Volume\n",
+    "\n",
+    "The quantum volume is wholistic quantum computer performance measure [QVOL0, QVOL1, QVOL2].\n",
+    "\n",
+    "\n",
+    "Roughly the logarithm (base 2) of quantum volume $V_Q$, quantifies the largest random circuit of equal width and depth that the computer successfully implements and is certifiably quantum.\n",
+    "\n",
+    "So if you have 64 qubits and the best log-QV you have measured is $\\log_2V_Q = 3$ the effectively you only have an 8 qubit device!\n",
```

Comment:
```suggestion
    "So if you have 64 qubits and the best log-QV you have measured is $\\log_2V_Q = 3$ then effectively you only have a 3 qubit device!\n",
```

Reply:
I'm not sure we should say "n qubit device" for any n, but regardless I think you meant n to be log-QV rather than QV. Otherwise an actual 3 qubit device could have log-QV equal to 3 and we would have said this is effectively a 8 qubit device.